### DISCUSSION TITLE
Need help with autodiscoverFilter

### DISCUSSION THREAD

**Discussion by mieliespoor** [renovatebot/renovate#16111] [Request Help] [answered]
We have a large number of projects in Bitbucket Server that we want to include some of them into Renovate. Given the list of projects below, how would me `autodiscoverFilter` param be looking like? I have tried a few examples, but can't seem to get it working.

```
https://bitbucket.link/projects/AWO
https://bitbucket.link/projects/CAT
https://bitbucket.link/projects/CRB
https://bitbucket.link/projects/EE
https://bitbucket.link/projects/CRDIA
https://bitbucket.link/projects/RTDVC
```

Lets say I want to include, `AWO`, `CRB`, `EE` and `RTDVC`?

**Comment by rarkins** [comment, marked as answer]
`autodiscoverFilter` can use regex so something like `/^(AWO|CRB|EE|RTDVC)/`. It's done on the repository name and not on the full URL like you have above

**Reply by mieliespoor** [reply]
Ok thanks. Will work with that then. We use bitbucket so the url of the repo then is `https://bitbucket.link/projects/{project-key}/repo/{repo-name}`. I was hoping to be able to filter on the project key to include repo's like that, but also use the negative lookup to exclude repo's in a similar manner.
